store.html

```diff
@@ -0,0 +1,166 @@
+         <div class="links">
+               <div class="logo">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 144 216">
+                        <path fill="#F15B2A"
+                            d="M126.1 212.9c4.1 4.1 10.7 4.1 14.8 0 4.1-4.1 4.1-10.7 0-14.8l-28.6-28.6 18.1-18.1c8.4-8.4 13.6-20 13.6-32.9C144 92.8 123.2 72 97.5 72S51 92.8 51 118.5c0 12.8 5.2 24.5 13.6 32.9l18.1 18.1-18.1 18.1c-4.6 4.6-11 7.5-18.1 7.5-14.1 0-25.6-11.5-25.6-25.6v-159C20.9 4.7 16.2 0 10.5 0 4.7 0 0 4.7 0 10.5v159C0 195.2 20.8 216 46.5 216c12.8 0 24.5-5.2 32.9-13.6l18.1-18.1 28.6 28.6zm-46.7-76.3c-4.6-4.6-7.5-11-7.5-18.1 0-14.1 11.5-25.6 25.6-25.6 14.1 0 25.6 11.5 25.6 25.6 0 7.1-2.9 13.5-7.5 18.1l-18.1 18.1-18.1-18.1z" />
+                        </svg>
+                    </div> 
+               
+            <a href="#"><strong>Meet Karma</strong></a>
+            <a href="#">How It Works</a>
+            <a href="#">Store</a>
+            <a href="#">Blog</a>
+            <a href="#">Help</a>
+            <a href="#">Login</a>
+          
+
+           <div class="hum"><svg width="24" height="16" viewBox="0 0 24 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H23C23.5523 0 24 0.447715 24 1V1.28572C24 1.838 23.5523 2.28572 23 2.28572H1C0.447716 2.28572 0 1.838 0 1.28572V1ZM0 7.85714C0 7.30485 0.447715 6.85713 1 6.85713H23C23.5523 6.85713 24 7.30485 24 7.85713V8.14285C24 8.69513 23.5523 9.14285 23 9.14285H1C0.447716 9.14285 0 8.69513 0 8.14285V7.85714ZM1 13.7143C0.447715 13.7143 0 14.162 0 14.7143V15C0 15.5523 0.447716 16 1 16H23C23.5523 16 24 15.5523 24 15V14.7143C24 14.162 23.5523 13.7143 23 13.7143H1Z" fill="black"/>
+                </svg></div> 
+      </div>
+  </nav>
+    <section class="section">
+        <form class="form" action="/action_page.php">
+          <h2 class="h2--store">Order your Karma wifi<br>divice today!</h2>
+                  <!-- note flex-->
+          <div class="parent--form">
+                
+            <div class="div--name"> 
+                    <label class="name" for="fname">Name*</label>
+                    <input type="text" class="name" name="fname" required>
+            </div>
+                    
+                   
+            <div class="div--name">
+                <label class="name" for="lname">Last Name*</label>
+                <input type="text" class="name" name="lname" required></div>
+            </div>
+                    
+            <label>Address*</label>
+            <input class="address" type="text" required><br>
+                    
+            <label>Address Line 2*</label>
+            <input class="address" type="text"><br>
+                    
+<!--note flex-->
+<div class="parent--form ">
+    <div class="div--name"> 
+        <label for="city">City*</label>
+        
+                    <select id="cities" name="city" form="city_form">
```

Comment:
The height of your City dropdown is different to the post code input. Can you please change this?